webapp/sources/ldap-inventory/inventory-repository/src/main/scala/com/normation/inventory/ldap/core/ReadOnlySoftwareInventoryDAOImpl.scala

```diff
@@ -173,8 +173,39 @@ class ReadOnlySoftwareDAOImpl(
     }
   }
 
+  def getNodesbySofwareName(softName: String): IOResult[List[(NodeId, Software)]] = {
+
```

Comment:
did you tried the corresponding `ldapQuery`? Is it worse in perf? (if so, we should try to make it better).

Reply:
Did i tried the request with ldapsearch and see how it behaves) or tried another ldap request ?

I make me think that i havent added the index yet in another pr

Reply:
the first one